readme.md

```diff
@@ -38,7 +38,7 @@ const run = async () => {
 
 Returns a `Promise` that is fulfilled when calling `input` returns a fulfilled promise. If calling `input` returns a rejected promise, `input` is called again until the maximum number of retries is reached. It then rejects with the last rejection reason.
 
-It doesn't retry on `TypeError` as that's a user error.
+It doesn't retry on `TypeError` as that's a user error. The only exclusion is fetchApi's 'Failed to fetch' error which can be a network error. See [the docs](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch), and [this issue](https://github.com/whatwg/fetch/issues/526)
```

Comment:
I think this sentence could be written better.

You also need to update index.d.ts

Reply:
Can you help me out with the wording? it seems fine for me

Reply:
May I suggest the following:

"It doesn't retry on `TypeError` as that's a user error. The only exclusion to this logic is when `TypeError` is thrown by `fetch`'s API with the message 'Failed to fetch', which [indicates that a request was not successful due to a network error](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful). However, beware that `fetch` may throw `TypeError` with different error messages on different platforms for similar situations. See https://github.com/whatwg/fetch/issues/526#issuecomment-554604080."

Better? 🤔

Reply:
ping @sindresorhus @oreporan

Reply:
Whatever @sindresorhus suggests as the text...Let me know

Reply:
What @acostalima wrote looks good to me.